easybuild/easyconfigs/r/Ruby-Tk/Ruby-Tk-0.2.0-intel-2018a-Ruby-2.5.1.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'RubyGem'
+
+name = 'Ruby-Tk'
+version = '0.2.0'
+
+rubyver = '2.5.1'
+versionsuffix = '-Ruby-%s' % rubyver
+
+homepage = 'https://github.com/ruby/tk'
+description = "Ruby/Tk"
```

Comment:
Maybe a more descriptive value like `"Ruby Tk interface module using tcltklib"` is better?

Reply:
@boegel Agreed.